src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2199,25 +2199,22 @@ def aks_get_versions(cmd, client, location):
 def aks_get_credentials(cmd, client, resource_group_name, name, admin=False,
                         path=os.path.join(os.path.expanduser(
                             '~'), '.kube', 'config'),
-                        overwrite_existing=False, context_name=None, public_fqdn=False):
+                        overwrite_existing=False, context_name=None, public_fqdn=False,
+                        credential_format=None):
     credentialResults = None
     serverType = None
     if public_fqdn:
         serverType = 'public'
+    if credential_format:
+        credential_format = credential_format.lower()
+        if admin:
+            raise InvalidArgumentValueError("--format can only be specified when requesting clusterUser credential.")
     if admin:
-        if serverType is None:
-            credentialResults = client.list_cluster_admin_credentials(
-                resource_group_name, name)
-        else:
-            credentialResults = client.list_cluster_admin_credentials(
-                resource_group_name, name, serverType)
+        credentialResults = client.list_cluster_admin_credentials(
```

Comment:
Could use `cmd.cli_ctx.cloud.profile` (of type string) to check the cloud profile.

Reply:
Switched to use it, thanks!